test/cli.py

```diff
@@ -533,6 +576,48 @@ def test_011_cli_benchmark(self):
 
         assert_success_of_builds([test_script], None, check_perf=True)
 
+    def test_012_cli_resume(self):
+
+        # NOTE: this is not a unit test, it relies on other command
+        # If this test is failing, make sure the following tests are passing:
+        # - test_cli_single
+        # - test_cli_build_dir
+
+        test_scripts = test_scripts_dot_py.keys()
+
+        # Build all the scripts, sending a crash signal to crash.py
+        with self.assertRaises(AssertionError):
+            testargs = [
+                "benchit",
+                "benchmark",
+                bash(f"{corpus_dir}/*.py"),
+                "--build-only",
+                "--cache-dir",
+                cache_dir,
+                "--script-args",
+                "crash!",
+            ]
+            with patch.object(sys, "argv", flatten(testargs)):
+                benchitcli()
```

Comment:
Add something here assuring that the test crashed as expected

Reply:
That's what the `self.assertRaises(AssertionError)` on line 589 is doing